<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import Photo to Edit</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Edited Photo</string>
   </property>
  </widget>
  <widget class="QSlider" name="redSlide">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="greenSlide">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="imageBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>321</width>
     <height>321</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="rgbRadio">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="grayscaleRadio">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Grayscale</string>
   </property>
  </widget>
  <widget class="QSlider" name="blueSlide">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sharpSlide">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="contrastSlide">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Default</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>~Placeholder~</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="redLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>53</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Red:</string>
   </property>
  </widget>
  <widget class="QLabel" name="greenLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>113</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Green:</string>
   </property>
  </widget>
  <widget class="QLabel" name="blueLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>173</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Blue:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sharpLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>235</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Sharpness:</string>
   </property>
  </widget>
  <widget class="QLabel" name="contrastLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>295</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Contrast:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorSuccess">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
